frontend/app/(general)/[username]/profile-content.tsx

```
@@ -0,0 +1,534 @@
+"use client";
+
+import React, { use, useState } from "react";
```

ESLint

[error] 'use' is defined but never used.

(@typescript-eslint/no-unused-vars)